client/src/pages/Reviews.js

```diff
@@ -95,10 +113,55 @@ const Reviews = () => {
     await getReviews(dispatch);
   };
 
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+    const updatedTab = { ...selectedTab };
+
+    if (newValue == 0) {
+      updatedTab.reviews = true;
+      updatedTab.requests = false;
+    } else {
+      updatedTab.reviews = false;
+      updatedTab.requests = true;
+    }
+    setSelectedTab(updatedTab);
+  };
+
   const handleListItemClick = (event, index, props) => {
     setSelectedIndex(index);
   };
 
+  const renderItems = (target) => {
+    console.log(state);
```

Comment:
remove this

Reply:
The console.log()? Yes, I missed that on my sweep through tidying up.